capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] Instead of using "upcase". kindly use "capitalize".
eg. `@nameable.correct_name.capitalize`

Reply:
Thank you very much for your review!